cdap-api/src/main/java/co/cask/cdap/api/ProgramConfigurer.java

```diff
@@ -44,4 +46,13 @@
    * @param properties the properties to set
    */
   void setProperties(Map<String, String> properties);
+
+  /**
+   * Sets the {@link RetryPolicy} to use when a retry-able remote failure is encountered.
+   * If none is set, the configured CDAP default values will be used. Different program types have different
+   * default retry policies. This policy can be overridden by preferences and runtime arguments.
+   *
+   * @param retryPolicy the retry policy to set
+   */
+  void setRemoteRetryPolicy(RetryPolicy retryPolicy);
```

Comment:
was trying to convey that it is for remote operations so that it doesn't sound like it's a policy for what should happen if the program fails. Open to changing the name though.